packages/RatPack.package/RPApplication.class/instance/updateFromRequest.with..st

```diff
@@ -1,9 +1,11 @@
 object handling
 updateFromRequest: aRouteSuffix with: anId
- 
-    | updatedObject jsonString aRPModelClass|
+
+	| updatedObject jsonString aRPModelClass objectFromDatabase |
```

Comment:
That's a lot of variables. I could condense `updatedObject` and `objectFromDatabase` into one called `object`. Also, we could in theory also remove `jsonString` since it doesn't add much readability. What do you think?

Reply:
With json String I agree.
The object thing I'd like to not change for this, because it shows the conflict of #106 
I renamed the method to ` getUpdatedObject: databaseObject withJson: ... `
because its not a call on the object itself.

I'd like to have after #106 a call like: ` object updateWithJson: ...` but for the moment I think its ok like that.

Reply:
Hmm?

```smalltalk
object := self database loadModelObject: (anId asNumber) from: aRPModelClass.
object := aRPModelClass updateObject: object object withJson: jsonString.
```
has the same ugliness, just one variable less, don't you think?

Reply:
We can do so, to me it’s unimportant.